src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -28,6 +28,7 @@
 import android.os.Build;
 import android.os.Bundle;
 import android.preference.PreferenceManager;
+import android.support.design.widget.Snackbar;
```

Comment:
Toast is still used in this case, not all the occurrences are safe to be replaced